setup_scripts/dagmc_setup.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#DAGMC Installation
+INSTALL_ROOT=$HOME
+cd $INSTALL_ROOT
+mkdir dagmc
+cd dagmc
+git clone https://github.com/svalinn/DAGMC
+cd DAGMC
+git checkout develop
+cd ..
+mkdir bld
+
+DAGMC_INSTALL_DIR=$INSTALL_ROOT/dagmc/install
+MOAB_CONFIG_DIR=$INSTALL_ROOT/MOAB/install/lib/cmake/MOAB
+mkdir $DAGMC_INSTALL_DIR
+cd bld
+cmake ../DAGMC -DCMAKE_INSTALL_PREFIX=$DAGMC_INSTALL_DIR -DMOAB_CMAKE_CONFIG=$MOAB_CONFIG_DIR
+make 
+make install
+
```

Comment:
It would be good to run the DAGMC tests here as well. This is done by entering the `test` directory in the DAGMC installation directory. All executables in that location can be run using `run-parts .`